### PR TITLE
Multiplayer initial

### LINKED ISSUE #17: As a multiplayer, I can see the opponent's blank screen board
#3 
Acceptance Tests: When going to /multi2, I can see 2 boards, one of which will be filled when a user types. 

A code can be generated, not shared yet

### PR DESCRIPTION
**Pull request by benpalay**
Closes #17

### CODE REVIEW

File: src/public/scripts/multiPlayerStart.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+// Function to generate random code
+function generateCode () {
```
Comment (IlanMalk) [inline src/public/scripts/multiPlayerStart.js:4]: I think the room code generation should ideally happen on the back end (which would also make it easier to unit test). This should be fine for now though, but we should address it in a later commit if time allows

File: src/public/scripts/multiPlayerStart.js [hunk 2]
```diff
@@ -28,15 +28,16 @@ codeButton.addEventListener('click', e => {
...
 })
 
 /* const startButton = document.getElementById('startGame')
-startButton.addEventListener('click', function () {
-  const code = document.getElementById('code').value
 
-}, false)
-*/
+startButton.addEventListener('click', e => {
+  e.preventDefault()
```
Comment (IlanMalk) [inline src/public/scripts/multiPlayerStart.js:41]: Perhaps we should remove this commented-out code?

**Review by IlanMalk — CHANGES_REQUESTED** [review]
Bug: 
On the multiplayer page, generate room code and click `OK`. An error results. Please fix this before the code can be accepted
image

Other than that, code is very clear, especially naming of variables, and conforms to StandardJS guidelines. Unit tests are not included although that is perhaps ok given that in terms of new functionality, only the generate room code function has been added. We might want to do this function on  the back end, both for testing and security purposes (though the CL could be approved without this and it could be done in a later sprint depending on time).

File: src/public/scripts/multiPlayerStart.js (hunk 1 above)
Comment (benpalay) [reply src/public/scripts/multiPlayerStart.js:4]: Agreed. For now it should be OK.

**Review by IlanMalk — APPROVED** [review]
The workaround prevents the bug so pull request approved

**Pull request merged**